Extension/src/LanguageServer/extension.ts

```diff
@@ -58,7 +58,11 @@ async function initVcpkgDatabase(): Promise<VcpkgDatabase> {
         const zip = new StreamZip.async({ file: util.getExtensionFilePath('VCPkgHeadersDatabase.zip') });
         try {
             const data = await zip.entryData('VCPkgHeadersDatabase.txt');
-            const lines = data.toString().split('\n');
+            // Strip '\r' and '\n' from the end of each line if they exist.
+            let lines = data.toString().split('\r\n');
+            if (lines.length === 1) {
```

Comment:
Do we not know what the line endings are?  We should own this file and how it's formatted.  If there is a way to split with a regex, you could split on `\r?\n` instead.

Reply:
There's a `splitLines` function in `Extension/src/common.ts`.